packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```diff
@@ -19,6 +17,10 @@
   visibility: hidden;
 }
 
+.option {
+  display: block;
```

Comment:
not a necessary change here. having the options be block makes it easier to fill up the vertical space and cause the content to flip to the other side when testing this out